e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-house-search.spec.js

```diff
@@ -0,0 +1,187 @@
+      companyDetails.companiesHouseSearch().should('have.focus');
+    });
+
+    it('should display the error in the input error summary', () => {
+      companyDetails.companiesHouseSearchError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${COMPANY_HOUSE_ERRORS[COMPANIES_HOUSE_INPUT_FIELD_ID].INCORRECT_FORMAT}`);
+        });
+    });
+  });
+
+  describe('when the companies house number has special characters', () => {
+    it('should display an error in the error summary', () => {
+      companyDetails.companiesHouseSearch().clear().type('123456!');
+      companyDetails.companiesHouseSearchButton().click();
+      partials.errorSummaryListItems().first().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(COMPANY_HOUSE_ERRORS[COMPANIES_HOUSE_INPUT_FIELD_ID].INCORRECT_FORMAT);
+        });
+    });
+
+    it('should focus to the input box when clicking the error', () => {
+      partials.errorSummaryListItemLinks().eq(0).click();
+      companyDetails.companiesHouseSearch().should('have.focus');
+    });
+
+    it('should display the error in the input error summary', () => {
+      companyDetails.companiesHouseSearchError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${COMPANY_HOUSE_ERRORS[COMPANIES_HOUSE_INPUT_FIELD_ID].INCORRECT_FORMAT}`);
+        });
+    });
+  });
+
+  describe('when the companies house number has a space', () => {
+    it('should display an error in the error summary', () => {
+      companyDetails.companiesHouseSearch().clear().type('123456 ');
+      companyDetails.companiesHouseSearchButton().click();
+      partials.errorSummaryListItems().first().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(COMPANY_HOUSE_ERRORS[COMPANIES_HOUSE_INPUT_FIELD_ID].INCORRECT_FORMAT);
+        });
+    });
+
+    it('should focus to the input box when clicking the error', () => {
+      partials.errorSummaryListItemLinks().eq(0).click();
+      companyDetails.companiesHouseSearch().should('have.focus');
+    });
+
+    it('should display the error in the input error summary', () => {
+      companyDetails.companiesHouseSearchError().invoke('text')
+        .then((text) => {
+          expect(text.trim()).equal(`Error: ${COMPANY_HOUSE_ERRORS[COMPANIES_HOUSE_INPUT_FIELD_ID].INCORRECT_FORMAT}`);
+        });
+    });
+  });
+
+  describe('when the companies house number is correctly entered', () => {
+    it('should not display errors', () => {
+      companyDetails.companiesHouseSearch().clear().type('8989898');
+      companyDetails.companiesHouseSearchButton().click();
+      companyDetails.errorSummary().should('not.exist');
+      companyDetails.companiesHouseSearchError().should('not.exist');
+    });
+
+    it('should display your business summary list', () => {
+      companyDetails.companiesHouseSearch().clear().type('8989898');
```

Comment:
Do we use the same CH number as DTFS? Just wonder if we're OK using this

Reply:
This is the one from the hint that is displayed so thought I should use it

Reply:
OK makes sense